scripts/process/process.storm_gage_height.R

```diff
@@ -1,6 +1,26 @@
-process.storm_gage_height <- function(viz){
-  # resample the gage data down to the same steps as timesteps
-  warning('need to resample gage data to `timesteps`')
+# resample the gage data down to the same steps as timesteps
+
+process.storm_gage_height <- function(viz = as.viz("storm-gage-height")) {
+  library(dplyr)
   deps <- readDepends(viz)
-  saveRDS(deps[['gage-height']], file = viz[['location']])
+  checkRequired(deps, c("gage-height", "timesteps"))
+  
+  times <- as.POSIXct(strptime(unlist(deps[["timesteps"]]),'%b %d %I:%M %p'), tz = "America/Puerto_Rico")
+  
+  gage_height <- deps[["gage-height"]]
+  attr(gage_height$dateTime, "tzone") <- "America/Puerto_Rico"
+  
+  gage_height <- gage_height %>% 
+    group_by(site_no) %>% 
+    filter(min(dateTime) >= times[2], max(dateTime) <= tail(times, 2L)[2]) %>%
```

Comment:
This is the opposite logic from the code I took this from because it filtered everything out with < and > rather than > and <... I'm confused.

Reply:
yeah, I see that. i don't get it either then.

Reply:
can you just remove this filter step?

Reply:
yep.